packages/jsii-reflect/lib/method.ts

```diff
@@ -109,4 +110,18 @@ export class Method implements Documentable, Overridable, TypeMember {
 
   public isMethod(): this is Method { return true; }
   public isProperty(): this is Property { return false; }
+
+  /**
+   * Return the location in the module
+   */
+  public get moduleLocation(): SourceLocation | undefined {
```

Comment:
The name suggests it'll return the location **of** the module, not **in** the module. I don't really like how this is named `moduleLocation` anyway, I'd rather have this named `sourceLocation`, really.

Reply:
I originally had it as `sourceLocation`, but I want the name to be clear about what it's relative to. In this case, it's a module-relative location.

It becomes relevant later when we're combining it with the assembly path to get a repository-relative location.

Reply:
I still think `moduleLocation` is misleading, _especially_ considering your argument.
You could name that `sourceCoordinates` (to me, "coordinates" feels more "relative to some referential") than "location")